lib/src/operations/cardano_derive_address_operation.dart

```diff
@@ -0,0 +1,72 @@
+import 'dart:typed_data';
+import 'package:buffer/buffer.dart';
+import 'package:ledger_cardano/src/operations/cardano_ledger_operation.dart';
+import 'package:ledger_cardano/src/operations/complex_ledger_operations.dart';
+import 'package:ledger_cardano/src/operations/ledger_operations.dart';
+import 'package:ledger_cardano/src/utils/cardano_networks.dart';
+import 'package:ledger_cardano/src/utils/hex_utils.dart';
+
+class CardanoDeriveAddressOperation extends ComplexLedgerOperation<String> {
+  final List<int> bip32SpendingPath;
+  final List<int> bip32StakingPath;
+  final CardanoNetwork network;
+
+  CardanoDeriveAddressOperation({
+    required this.bip32SpendingPath,
+    required this.bip32StakingPath,
+    required this.network,
+  });
+
+  @override
+  Future<String> invoke(LedgerSendFct send) async {
+    final ByteDataWriter writer = ByteDataWriter();
+
+    writer.writeUint8(0);
+    writer.writeUint8(network.networkId);
+
+    appendSpendingDataSource(writer, bip32SpendingPath);
+    appendStakingDataSource(writer, bip32StakingPath);
+
+
+    final response = await send(
+      SendOperation(
+        ins: InstructionType.deriveAddress.insValue,
+        p1: ReturnType.returnData.p1Value,
+        p2: 0,
```

Comment:
Same here. please add at least a comment to say "0 means unused"

Reply:
done